### PR TITLE
Update ome-faq.yml

### PR DESCRIPTION
**Pull request by samschan-ms**
@KCCross  please review and merge

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/ome-faq.yml [hunk 1]
```diff
@@ -191,7 +191,7 @@ sections:
       - question: |
           Can I use message encryption with compliance features such as eDiscovery?
         answer: |
-          Yes. All encrypted email messages are discoverable by Microsoft 365 compliance features.
+          Yes. All encrypted email messages are discoverable by Microsoft 365 compliance features.  See [eDiscovery activities tha that suppport encrypted items](https://docs.microsoft.com/microsoft-365/compliance/ediscovery-decryption?view=o365-worldwide#ediscovery-activities-that-support-encrypted-items) for details.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ome-faq.yml]:
```suggestion
          Yes. All encrypted email messages are discoverable by Microsoft 365 compliance features.  See [eDiscovery activities that support encrypted items](/microsoft-365/compliance/ediscovery-decryption?view=o365-worldwide#ediscovery-activities-that-support-encrypted-items) for details.
```

File: microsoft-365/compliance/ome-faq.yml [hunk 2]
```diff
@@ -247,8 +247,8 @@ sections:
       - question: |
           Can I open encrypted messages sent to another user's mailbox with Fullaccess?
         answer: |
-          Users can open encrypted messages as long as they are given direct access and automapping is turned ON. Access is not allowed if the access is granted via an email-enabled security group.
-
+          No, users can open encrypted messages in shared mailbox.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ome-faq.yml]:
```suggestion
          No, users can open encrypted messages in a shared mailbox.
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review - @samschan-ms, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks

**Pull request merged**